integration_tests/test_basic.py

```diff
@@ -103,15 +103,14 @@ def test_native_call(cronos):
 
     amount = 100
 
-    # the coinbase in web3 api is the same address as the validator account in
-    # cosmos api
+    tx = contract.functions.test_native_transfer(amount).buildTransaction(
+        {"from": ADDRS["validator"]}
+    )
+    receipt = send_transaction(w3, tx)
 
     # expect failure, because contract is not connected with native denom yet
     # TODO complete the test after gov managed token mapping is implemented.
-    with pytest.raises(ValueError):
```

Comment:
the error is not an exception anymore, but a zero receipt status now. I guess it's because error message changed.